index.html

```diff
@@ -213,8 +243,8 @@ <h3>
         The steps for <dfn>processing <a>ImageResource</a> members</dfn> are
         given by the following algorithm. The algorithm takes an
         Array&lt;<a>ImageResource</a>&gt; <var>entries</var> and a <a>URL</a>
-        <var>context</var>. This algorithm returns an
-        Array&lt;<a>ImageResource</a>&gt;.
+        <var>context</var>. This algorithm returns an Array&lt;<a>processed
```

Comment:
I think we should drop the `Array<>` notation here, since we're dealing with a "spec-level" object not a JS object, it doesn't make sense to say `Array<>` of one of these high-level objects.

In fact we probably shouldn't be using `Array<>` at all, since that isn't part of WebIDL or Infra (this part was inherited from very old text in Manifest). So many change the previous sentence too.

I think you can just say either "`sequence<ImageResource>`" (referencing the WebIDL [sequence type](https://heycam.github.io/webidl/#idl-sequence)) or "[list](https://infra.spec.whatwg.org/#list) of `ImageResource`s" (referencing the Infra [list concept](https://infra.spec.whatwg.org/#list)). For this sentence, "[list](https://infra.spec.whatwg.org/#list) of processed image resources".